windows-installer/setup.nsi

```diff
@@ -33,6 +33,9 @@
 ; Directory to install all files into.  This makes uninstall easier and safer.
 !define MAIN_INSTDIR "$INSTDIR\Main"
 
+; Directory for the user to add custom configs.
+!define CUSTOM_CONFIG_DIR "$INSTDIR\config.d"
```

Comment:
I should have been clearer. Let's look at a concrete scenario: suppose at some future point we decide to move `fluentd.conf` under `$INSTDIR\etc`, to separate it out from other files in that directory. The way the installer code is now, whoever does it will have to remember to both change `FLUENTD_CONFIG_LOCATION` and update the `CreateDirectory` command in line 193. If they forget and only update the former, the config will break. That's why I proposed defining `FLUENTD_CONFIG_DIRECTORY` and using it in both places instead of `$INSTDIR`.